### PR TITLE
cgroup: systemctl: Don't display NULL if protection was set to max

### PR DESCRIPTION
**Pull request by cdown**
Inside format_bytes, we return NULL if the value is UINT64_MAX. This
makes some kind of sense where this has some other semantic meaning than
being a value, but in this case the value is both a.) not the default
(so we definitely want to display it), and b.) means "infinity" (or
"max" in cgroup terminology).

This patch adds a small wrapper around format_bytes that can be used for
these cases, to avoid the following situation:

    [root@tangsanjiao ~]# cat /sys/fs/cgroup/workload.slice/memory.low
    max
    [root@tangsanjiao ~]# systemctl show workload.slice -p MemoryLow
    MemoryLow=infinity
    [root@tangsanjiao ~]# systemctl status workload.slice | grep low:
	Memory: 14.9G (low: (null))

After the patch:

    [root@tangsanjiao ~]# systemctl status workload.slice | grep low:
	Memory: 15.1G (low: infinity)

### CODE REVIEW

**Comment by cdown** [comment]
Adding to v245 since this caused some confusion internally and should be trivial enough.

**Comment by cdown** [comment]
test-exec-deserialization.py is unrelated, it has nothing to do with these changes.

File: src/basic/format-util.h [hunk 1]
```diff
@@ -71,8 +72,15 @@ typedef enum {
         FORMAT_BYTES_TRAILING_B  = 1 << 2,
 } FormatBytesFlag;
 
-#define FORMAT_BYTES_MAX 8
+#define FORMAT_BYTES_MAX 16
```
Comment (keszybz) [inline src/basic/format-util.h:75]: Why this change?

Comment (cdown) [reply src/basic/format-util.h:75]: "infinity\0" is 9 bytes. We can special case it for this one case, but doing so would make it harder to reuse buffers between `format_bytes_*` which kind of sucks. I can also change it to be 9, but I shuddered a bit at not being power-of-two aligned.

We could also just put "inf", but it probably makes sense to output something also accepted by the parser.

Since we probably aren't going to have tons of these sitting in memory at once, seemed like a reasonable thing to do to just up it.

Comment (keszybz) [reply src/basic/format-util.h:75]: OK.
